snooty/diagnostics.py

```diff
@@ -29,15 +29,14 @@ def __init__(
         self.end = (end_line, end_column)
 
     class Level(enum.IntEnum):
-        info = 0
-        error = 1
+        info = 1
         warning = 2
+        error = 3
```

Comment:
Is there a reason why the internal `Level` numbering is the opposite of standard (`error=1`, etc...)?

Reply:
This matches the internal doctutils diagnostic code scheme, but that's mostly an arbitrary decision! It is however more common for increasingly severe error conditions to have higher values (see: Python, Java standard libraries). Language Server Protocol is the odd one out, as far as I'm aware.